docs/proposals/multi-version.md

```diff
@@ -0,0 +1,53 @@
+# Multi-Version-Landscaper-Instances
+
+This document contains a proposal how to support multiple versions of Landscaper Instances in one Landscaper as a 
+Service (LaaS) landscape.
+
+## Requirements
+
+- A LaaS landscape should support different versions of a Landscaper instance (LI), i.e. different LandscaperDeployments 
+  install different versions of a [this](../../.landscaper/landscaper-instance) component. A more detailed
+  selection of particular deployer versions seems to be too complicated and out of scope.
+
+- A LaaS landscape supports a predefined set of LI versions, which change from release to realease (probably if the LaaS 
+  component version changes).
+
+- The Central Landscaper should always be on the latest Landscaper version. The version referenced by the 
+  LaaS project.  
+
+- A customer must be able 
+  - to select its intended LI version during onboarding. This must be optional. He gets the latest version
+    if nothing is selected.
+  - to upgrade to its intended LI version. Therefore, an allowed upgrade path must be enforced.
+  - see which versions are currently supported, which are recommended and which are outdated and what are the possible
```

Comment:
And a customer must be able to see the exact timeframes in which
- specific LI versions are supported
- specific LI versions are set to "deprecated"
- specific LI versions run out of support -> results in a forced-upgrade ?
-

Reply:
I have included these requirements